pages/linux/goldeneye.md

```diff
@@ -0,0 +1,24 @@
+# goldeneye
+
+> A HTTP DoS test tool.
+> More information: <https://github.com/jseidl/GoldenEye>.
+
+- Test a specific website:
+
+`./goldeneye.py {{url}}"`
+
+- Test a specific website with 100 user agents and 200 concurrent sockets:
```

Comment:
```suggestion
- Test a specific website with 100 [u]ser agents and 200 concurrent [s]ockets:
```

Reply:
- Mnemonics are redundant for long options.
- Such case is not handled by our style guide.